example-gto/generate.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+
+# Setup script env:
+#   e   Exit immediately if a command exits with a non-zero exit status.
+#   u   Treat unset variables as an error when substituting.
+#   x   Print commands and their arguments as they are executed.
+set -eux
+
+PUSH=false
+echo $#
+if [ "$#" -eq 1 ] && [ "$0" != "--push" ]; then
+   PUSH=true;
+   echo "Will push things to GitHub :tada:"
+fi
+
+HERE="$( cd "$(dirname "$0")" ; pwd -P )"
+USER_NAME="iterative"
+REPO_NAME="example-gto"
+
+BUILD_PATH="$HERE/build"
+
+mkdir $BUILD_PATH
+pushd $BUILD_PATH
+if [ ! -d "$BUILD_PATH/.venv" ]; then
+  virtualenv -p python3 .venv
+  export VIRTUAL_ENV_DISABLE_PROMPT=true
```

Comment:
what does it do?

Reply:
https://virtualenv.pypa.io/en/16.7.9/reference.html#envvar-VIRTUAL_ENV_DISABLE_PROMPT

Reply:
sorry, I meant to ask - is there a specific reason to have it here? (it can be error prone if you see that venv is not activated)

Reply:
Don't know TBH, just copied this over from mlem PR. Let's delete it :)